src/ServiceControl/Recoverability/Retrying/RetryingManager.cs

```diff
@@ -85,7 +85,7 @@ public async Task Forwarding(string requestId, RetryType retryType)
                 return;
             }
 
-            var summary = Get(requestId, retryType);
```

Comment:
No idea why this was `Get` instead of `GetOrCreate`. I looked at the invocation path and it seemed like it could (after SC restart) be invoked without prior call to `GetOrCreate` so I changed it to `GetOrCreate` to be safe.